.travis/install.sh

```diff
@@ -7,7 +7,7 @@ sudo add-apt-repository -y ppa:costamagnagianfranco/borgbackup
 sudo apt-get update
 sudo apt-get install -y fakeroot borgbackup
 
-pip install 'virtualenv<14.0'
+pip install 'virtualenv==16.7.7'
```

Comment:
in the requirements file, you removed the version restriction.

is that also possible here?